src/ray/exported/exported.cc

```diff
@@ -0,0 +1,34 @@
+#include "ray/core_worker/core_worker.h"
+#include "ray/exported/test.h"
+
+namespace ray {
+namespace exported_ {
```

Comment:
Can we name this differently? "exported" is not very descriptive. It would be good if the name can include "streaming" to indicate what the exports are for. Also, please add a description in the comments on the header file to explain.

Reply:
Conceptually, any lib that will directly use the C++ core worker API should use this. So I think it's better to not add "streaming" in the namespace. 
But we should add a comment in the header file to explain why this is needed. Especially, we should say that it resolves the symbol conflicting issue from the streaming lib.